docs/index.md

```diff
@@ -19,4 +19,5 @@ description: |-
 
 - `host` (String) Target to tsuru API
 - `skip_cert_verification` (Boolean) Disable certificate verification
+- `skip_managed_by_validation` (Boolean) Disable ManagedBy Validation when read resources (optional)
```

Comment:
@jbragagnolo you could rename this to a name that describe that behavior is just for env resources.

Reply:
@wpjunior Renamed parameter and description to: 
- skip_envs_managed_by_tsuru: (Boolean) Skip environment variables managed by Tsuru (optional)

What do you think?

Reply:
I don't know, that name still doesn't sound good to me.
Some suggestion?

Reply:
@jbragagnolo I tough in some names:

take_over_environment_variables
assume_all_environment_variables
full_management_of_environment_variables

Reply:
How about user_management_of_environment_variables?
Because when we think about all or full, we should take all types available in managed by.

Reply:
I understand @hugoledra, so a rename parameter to `full_management_of_user_environment_variables`.